tasks/e2e/cypress/integration/03-storybook/storybook.spec.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-undef, camelcase */
+/// <reference types="cypress" />
+
+const BASE_DIR = Cypress.env('RW_PATH')
+
+describe('Redwood Storybook Integration', () => {
+  it('0.1 Build Storybook Static Files', () => {
+    cy.exec(`cd ${BASE_DIR}; yarn rw storybook --build`, {
+      timeout: 50000,
+    })
+      .its('code')
+      .should('eq', 0)
+  })
+
+  it('1. BlogPost Component', () => {
+    cy.visit(
+      `http://localhost:8910/storybook-static/iframe.html?id=components-blogpost--generated&viewMode=story`
+    )
+    // TO DO: code mode to update mock
+    cy.get('#error-message').should('contain.text', 'Cannot read property')
+  })
+
+  it('2. BlogPostCell', () => {
+    cy.visit(
+      `http://localhost:8910/storybook-static/iframe.html?id=cells-blogpostcell--loading&viewMode=story`
```

Comment:
Should now be `http://localhost:8910/storybook`. Also, would be great to set this via config as the base URL for all checks.